models/generics/contact_information.py

```diff
@@ -63,6 +59,12 @@ class Meta:
         """
 
         verbose_name_plural = 'contact information'
+        constraints=[
+            models.UniqueConstraint(fields=['entity_content_type', 'primary_phone_number'], name='unique_primary_number'),
+            models.UniqueConstraint(fields=['entity_content_type', 'secondary_phone_number'], name='unique_secondary_number'),
+            models.UniqueConstraint(fields=['entity_content_type', 'email_address'], name='unique_email_address'),
+            models.UniqueConstraint(fields=['entity_content_type', 'institute_webmail_address'], name='unique_institute_webmail_address'),
+     ]
```

Comment:
Closing bracket not aligned with opening line.
```suggestion
        constraints=[
            models.UniqueConstraint(fields=['entity_content_type', 'primary_phone_number'], name='unique_primary_number'),
            models.UniqueConstraint(fields=['entity_content_type', 'secondary_phone_number'], name='unique_secondary_number'),
            models.UniqueConstraint(fields=['entity_content_type', 'email_address'], name='unique_email_address'),
            models.UniqueConstraint(fields=['entity_content_type', 'institute_webmail_address'], name='unique_institute_webmail_address'),
        ]
```

Reply:
Also all these lines seem to be way over the 80 character limit.